src/geo/io.clj

```diff
@@ -23,4 +25,4 @@
 (defn to-wkb [^Geometry geom] (.write wkb-writer geom))
 
 (defn read-geojson [^String geojson] (.read geojson-reader geojson))
-(defn to-geojson [^Geometry geom] (.toString (.write geojson-writer geom)))
+(defn to-geojson [^Geometry geom] (.toString (.write geojson-writer geom)))
```

Comment:
Small quibble but it looks like the newline at the end of this file got chomped...can we add it back?